state/runtime/evm/instructions.go

```diff
@@ -907,6 +907,17 @@ func opGasLimit(c *state) {
 	c.push1().SetInt64(c.host.GetTxContext().GasLimit)
 }
 
+func opBaseFee(c *state) {
+	if !c.config.London {
+		c.exit(errOpCodeNotFound)
+
+		return
+	}
+
+	baseFee := new(big.Int).Set(c.host.GetTxContext().BaseFee)
+	c.push1().SetUint64(baseFee.Uint64())
```

Comment:
A small suggestion if you want to avoid the allocation.
```suggestion
	c.push(c.host.GetTxContext().BaseFee)
```

Reply:
Done